contrib/client/teos_client.py

```diff
@@ -463,6 +499,18 @@ def main(command, args, command_line_conf):
                 if appointment_data:
                     print(appointment_data)
 
+        elif command == "get_subscription_info":
+            if args:
+                arg_opt = args.pop(0)
+
+                if arg_opt in ["-h", "--help"]:
+                    sys.exit(help_get_subscription_info())
+
+            teos_id = load_teos_id(config.get("TEOS_PUBLIC_KEY"))
+            subscription_info = get_subscription_info(user_sk, teos_id, teos_url)
+            if subscription_info:
+                print(subscription_info)
```

Comment:
It may be worth pretty printing this (this may also apply to `appointment_data`). This should also use logger.info instead of print.

Feel fee to leave this as a followup.

Reply:
BTW by pretty printing I mean `json.dumps(x, indent=4)`.

We do so in the CLI but looks like we forgot to port it here too.